.github/workflows/performance-testing.yaml

```diff
@@ -76,9 +83,11 @@ jobs:
           JSONTAG=$(git show -s --format="%H")
           docker pull ${{ secrets.ECR_REPOSITORY }}/stargateio/${{ matrix.docker-image }}:$JSONTAG
           docker image tag ${{ secrets.ECR_REPOSITORY }}/stargateio/${{ matrix.docker-image }}:$JSONTAG stargateio/${{ matrix.docker-image }}:$JSONTAG
+          SGTAG="$(./mvnw -f . help:evaluate -Dexpression=stargate.int-test.coordinator.image-tag -q -DforceStdout)"
```

Comment:
In copying steps from docker pull from this performance testing workflow for the new postman workflow, I realized that this workflow wasn't actually pulling the coordinator image from ECR. I updated both workflows with logic to pull the coordinator image according to the Stargate version referenced in the pom.